homeassistant/components/swidget/coordinator.py

```diff
@@ -0,0 +1,54 @@
+"""Swidget Device Coordinator."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from swidget import SwidgetDevice, SwidgetException
+
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.debounce import Debouncer
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+REQUEST_REFRESH_DELAY = 1.0
+
+
+class SwidgetDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
Please type the DataUpdateCoordinator with the return type of async update data 

```suggestion
class SwidgetDataUpdateCoordinator(DataUpdateCoordinator[..]):
```

Reply:
Updated